versioned_docs/version-v4.0.0/developers/intro.md

```
@@ -0,0 +1,51 @@
+id: intro
+title: Developing HOPR apps
+group: h-no-count
+---
+
+The HOPR network is only useful if data is being transferred through it. In fact, the more data transported
+through HOPR nodes, the more private the whole network becomes. The HOPR network is designed to have constant
+traffic, making it impossible for malicious actors to subtract meaningful information from it.
+
+HOPR apps are a key component in providing this flow of data. Developers can build applications on top of the HOPR network to
+provide users with a private channel for exchanging information. At the same time, HOPR apps create traffic within
+the network, increasing the amount of noise mixers used to protect the entire ecosystem. You can learn more about the HOPR protocol [here.](https://medium.com/hoprnet/basics/home).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)